libs/coin-framework/src/derivation.ts

```diff
@@ -182,6 +182,9 @@ const modes: Readonly<Record<DerivationMode, ModeSpec>> = Object.freeze({
   ton: {
     overridesDerivation: "44'/607'/0'/0'/<account>'/0'",
   },
+  sui: {
+    overridesDerivation: "44'/784'/0'/0'/0'",
```

Comment:
I think we should put `<account>` placeholder somewhere. Probably at the last position.

Reply:
Possible. According to documentation https://docs.sui.io/concepts/cryptography/transaction-auth/keys-addresses it should be `44'/784'/<account>'/0'/0'`

Reply:
Confirmed. It's the proper way and Sui Wallet uses it